src/rviz/image/image_display_base.cpp

```diff
@@ -65,6 +65,18 @@ ImageDisplayBase::ImageDisplayBase() : Display(), sub_(), tf_filter_(), messages
 
   unreliable_property_ = new BoolProperty("Unreliable", false, "Prefer UDP topic transport", this,
                                           &ImageDisplayBase::updateTopic);
+
+  reset_to_property_ = new BoolProperty("Reset When Timed Out", false,
+                                        "Reset when new image has not been received for Timeout [sec].",
+                                        this, &ImageDisplayBase::updateResetTO);
```

Comment:
Please drop this property and encode no timeout via `timeout_property_ = 0`.
This is to avoid cluttering the property interface and is frequently used in similar situations.